README.md

```diff
@@ -13,4 +13,4 @@ one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,
 
 ## Proof:
 
-For a graph to be completely connected each vertex must have an edge that leads to every other vertex which means that every vertex in the graph must be identical to every other vertex, because every vertex has the same number of edges. Since we know that $A$ and $B$ have the same number of nodes and we know that each vertex within both of those graphs has the same number of edges, then any order of vertexes in $A$ can be mapped to any order of vertexes in $B$ and because these graphs are fully connected this will be a proper, isomorphic mapping.
+For a graph to be completely connected each vertex must have an edge that leads to every other vertex which means that every vertex in the graph must be equivalent to every other vertex, because every vertex has the same number of edges. Since we know that $A$ and $B$ have the same number of nodes and we know that each vertex within both of those graphs has the same number of edges, then any order of vertexes in $A$ can be mapped to any order of vertexes in $B$ and because these graphs are fully connected this will be a proper, isomorphic mapping.
```

Comment:
Can you refer to the definition explicitly here and say what parts are satisfied by what parts of your argument?